Makefile

```diff
@@ -29,9 +29,10 @@ tidy:
 
 # Install the tools for local development.
 install-tools:
-	go install github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen@latest
+	go install github.com/oapi-codegen/oapi-codegen/v2/cmd/oapi-codegen@latest
 	go install github.com/sqlc-dev/sqlc/cmd/sqlc@latest
 	go install ariga.io/atlas/cmd/atlas@latest
+	go install github.com/cosmtrek/air@v1.40.4
```

Comment:
Just curious here.. is there a particular reason for going for the specific version of `v1.40.4` as opposed to the newest one that's available? Wasn't sure if there was

---

This reminds me though - it likely makes sense for all of these tools under the `install-tools` target to eventually have a specific version, as opposed to just `@latest`. Mostly, since we likely want everyone to use the same version of the tool when running them, so everyone gets the same result.

This can be done later though as tools start getting used, e.g. once we start using `sqlc` to generate the code for the DB queries, we should then also change `sqlc@latest` with the specific version being used.